### ISSUE TITLE
lpgen.js don't put some icon names to langpack

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#1500]
![42724096-ce2edb5a-8774-11e8-9c4e-a80efb3c0e9a](https://user-images.githubusercontent.com/5046855/42738000-08ca2628-8885-11e8-9a51-dcd8b386b69c.png)

[23:11]	dartraiden: этот значок берёт текст из пункта меню
[23:12]	dartraiden: но, в меню запись выглядит как Hide &offline users out here
[23:12]	dartraiden: и для нее есть перевод
[23:12]	dartraiden: а вот в текст значка попадает строка без &
[23:12]	dartraiden: но lpgen.js про это не знает

**Comment by dartraiden** [comment]
В "меню групп" тоже есть такой значок.

**Comment by dartraiden** [comment]
У одного значка TabSRMM такая же история

![42724094-c98e364a-8774-11e8-9c4d-fc4af5181334](https://user-images.githubusercontent.com/5046855/42738035-b4099d70-8885-11e8-8293-859e33812eba.png)

[&Send later job list...]
Задачи для &отправки с задержкой...

но в описании значка Миранда сама вырезает &, что приводит к отсутствию перевода

**Comment by tweimer** [comment]
It doesn't look good to use & in icon descriptions anyways

**Comment by dartraiden** [comment]
Yes, so solution - give icon own name, not obtain name from menu item
